targets/gimbal/include/PinNames.h

```diff
@@ -143,78 +143,53 @@ typedef enum {
   ADC_VREF = 0xF1,
   ADC_VBAT = 0xF2,
 
-// Orson TODO: everything below this you should be fixing and working on. make sure all the pins are aliased correctly.
-// that is that what is on the ioc is on here. ask Ari to share what he found was wrong and you fix it. Work closely
-// with Aidan Gratton and keep him in the loop with what you're doing STDIO for console print
-// once you are done make sure every single pin here is defined in the ioc and that the every single pin on the ioc is
-// here
 #ifdef MBED_CONF_TARGET_STDIO_UART_TX
   STDIO_UART_TX = MBED_CONF_TARGET_STDIO_UART_TX,
 #else
-  STDIO_UART_TX =
-      PC_10,  // Orson TODO: I think this should be NC bc of new flashing and printing method. check with Cindy
+  STDIO_UART_TX = PC_10,
 #endif
 #ifdef MBED_CONF_TARGET_STDIO_UART_RX
   STDIO_UART_RX = MBED_CONF_TARGET_STDIO_UART_RX,
 #else
-  STDIO_UART_RX = PC_11,  // Orson TODO: same as above
+  STDIO_UART_RX = PC_11,
 #endif
 
   /**** UI ****/
-  // Orson TODO: name these based on their color. mbed will complain if you dont define LED1 so define one of em to be
-  // LED1
-  LED1 = PC_0,
-  LED2 = PC_1,
-  LED3 = PC_2,
-  LED4 = PC_3,
-  // Orson TODO: led matrix is not on gimbal so delete
-  LED_MTRX = PA_2,
-
-  BUTTON_1 = PB_0,
-  BUTTON_2 = PB_1,
+  LED1  = PC_7,
+  LED_G = PA_0,
+  LED_B = PA_1,
+  LED_R = PA_2,
+
+  BUTTON_1 = PA_3,
+
+  /*** pins needed for other apps to work ***/
+  LED2 = LED_R,
 
   /**** Actuator Pins ****/
-  // Orson TODO: name these better for example. SRVO_PWM_PAN should be the pwm pin for controlling Pan Servo
-  SRVO_PWM_CR = PB_14,
-  SRVO_PWM_HS = PB_15,
-  SRVO_PWM_SG = PB_5,
-
-  /**** Limit Switch Pins ****/
-  // Orson TODO: figure out what has limit switches and if we can add them see arm app for how we deal with limit
-  // switches
-  LIM_GIMB = PC_13,
-
-  /**** Encoder Pins ****/
-  // Orson TODO: figure out what this is for and name it better. actually this prob goes away since i think we use
-  // aeat6012 on gimbal not mae3
-  ENC_PWM_GIMB = PA_3,
-
-  /**** Sensors ****/
-  // Orson TODO: remove these idk we have US on gimbal
-  ULTRA_TRIG_3V3_1 = PB_6,
-  ULTRA_ECHO_1     = PB_7,
-  ULTRA_TRIG_3V3_2 = PB_8,
-  ULTRA_ECHO_2     = PB_9,
+  SRVO_PWM_YAW   = PB_6,
+  SRVO_PWM_PITCH = PB_4,
+  SRVO_PWM_ROLL  = PB_5,
 
   /**** Serial Interface Pins ****/
+  SPI1_CS   = PC_4,
   SPI1_SCK  = PA_5,
   SPI1_MOSI = PA_7,
   SPI1_MISO = PA_6,
 
-  CAN1_TX = PA_12,
-  CAN1_RX = PA_11,
+  CAN_TX_A = PA_12,
+  CAN_RX_A = PA_11,
 
-  CAN2_TX = PB_13,
-  CAN2_RX = PB_12,
+  CAN_TX_B = PB_13,
+  CAN_RX_B = PB_12,
 
-  CAN_TX = CAN1_TX,
-  CAN_RX = CAN1_RX,
+  CAN_TX = CAN_TX_A,
+  CAN_RX = CAN_RX_A,
 
-  SERIAL_TX  = STDIO_UART_TX,
-  SERIAL_RX  = STDIO_UART_RX,
-  SERIAL_RTS = PC_8,
-  // does not exist
-  // Orson TODO: figure out what the above comment was for. what doesnt exist? if it doesnt exist delete it
+  CAN1_TX = CAN_TX,
+  CAN1_RX = CAN_RX,
```

Comment:
What's up with having CAN A, CAN B, CAN, and CAN 1? Is it just to support the different naming conventions in test apps?

Reply:
yeah dont change the CAN names leave em as CAN1 and CAN2 and CAN

Reply:
@cindyli-13 It was from the IOC file given to me by electrical. @younesr1 said to make sure the names in the IOC file match the ones in the gimbal PinNames.h. Also for compatibility with other apps I did alias some of the pins to have them work.

Reply:
yeah I'd say using CANA and CANB would only cause confusion, so i think its best to stick to the convention in the rest of the repo. ig this is a bit of a nit pick so idk if u wanna share with ee person who made this ioc

Reply:
You can remove CAN_A and CAN_B and just have CAN, CAN1, CAN2